docs/gadget-devel/gadget-ebpf-api.md

```diff
@@ -341,7 +341,7 @@ Symbolize the kernel stack from `gadget_get_kernel_stack(ctx)` (see [kernel-stac
 
 ### `gadget_user_stack`
 
-Symbolize the user stack from `gadget_get_user_stack(ctx, &event->ustack_raw)` (see [user-stack-traces](#user-stack-traces)).
+Symbolize the user stack from `gadget_get_user_stack(ctx, &event->ustack)` (see [user-stack-traces](#user-stack-traces)).
```

Comment:
Why did you remove the `_raw` here? The main reason we opted for fields with a `_raw` suffix was to not have overlapping field definitions that vary in their types depending on whether an operator was active or not. In this case, without the operator the type will be a blob of addresses, but with the operator it will become an `object` (which doesn't have that blob anymore in the currently supported output modes) with two new attributes `addresses` and `symbols`, right?

Reply:
In both cases (with or without the operator), ustack is a struct. The operator adds some subfields in the struct but does not overwrite the type of any subfield.
I can reintroduce separate ustack and ustack_raw but it is a bit more complicated to create ustack because I then need to create both the ustack outer struct and the inner subfields.

Reply:
Ah, sorry, makes sense. Let's keep it without the `_raw`, then.